source/monsters/monsters_a-z/A/index.rst

```diff
@@ -5,5 +5,5 @@ A
    :titlesonly:
    :glob:
 
-   */index
+   **/index
```

Comment:
Why this change?

Side note: we should really standardize how we do this. It's different in a bunch of places. We could probably just stick `**` in here and be good.